geode-core/src/main/java/org/apache/geode/management/internal/cli/remote/CommandExecutor.java

```diff
@@ -109,6 +109,9 @@ protected Object invokeCommand(Object command, GfshParseResult parseResult) {
     SingleGfshCommand gfshCommand = (SingleGfshCommand) command;
     ResultModel resultModel = (ResultModel) result;
     if (resultModel.getStatus() == Result.Status.ERROR) {
+      InfoResultModel errorInfo = resultModel.addInfo();
```

Comment:
I think we probably don't need this. If the command already failed. No one would be expecting the cluster config be changed.

Reply:
Wasn't so sure of it myself.  I'll pull it out.